fix: miniature of ingredients blocks the text

M123-dev opened the pull request:
### What

- Added a with to the image preview, now it's a square
- Fixes: #2959
- I'm currently not able to provide a screenshot due to #2963

M123-dev committed:
fix: miniature of ingredients blocks the text

M123-dev committed:
Update ml_kit_scan_page.dart

teolemon committed:
Merge branch 'develop' into fix-miniature-of-ingredients-blocks-the-text

monsieurtanuki commented on packages/smooth_app/lib/pages/product/add_new_product_page.dart:
```diff
@@ -189,11 +189,17 @@ class _AddNewProductPageState extends State<AddNewProductPage> {
         mainAxisAlignment: MainAxisAlignment.center,
         children: <Widget>[
-          SizedBox(height: 50, child: Image.file(image, fit: BoxFit.cover)),
+          SizedBox(
+            height: MINIMUM_TOUCH_SIZE,
+            width: MINIMUM_TOUCH_SIZE,
+            child: Image.file(image, fit: BoxFit.cover),
+          ),
```
That's not that big. Probably ridiculous for images in height.

monsieurtanuki reviewed (approved):
Hi @M123-dev!
Not very convinced: of course it probably solves the issue (a screenshot would be nice), but the Image is therefore probably a bit little.

Possible improvement: use the RRect like in the product items in lists.
Not sure if it's much bigger, but at least it's vaguely consistent with the rest of the app.

M123-dev committed:
Update add_new_product_page.dart

M123-dev committed:
Update add_new_product_page.dart

M123-dev commented:
Thanks for the review @monsieurtanuki, I added the RRect and changed the size back to 50 as this will be changed anyways by https://github.com/openfoodfacts/smooth-app/issues/2843

M123-dev committed:
Merge branch 'develop' into fix-miniature-of-ingredients-blocks-the-text
